.gitlab-ci.yml

```diff
@@ -301,6 +308,114 @@ dogstatsd_suse-x64:
     paths:
       - $OMNIBUS_PACKAGE_DIR_SUSE
 
+# deploy debian packages to apt staging repo
+deploy_deb_testing:
+  stage: deploy_testing
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/datadog-agent-builders/deploy:latest
+  before_script:
+    - ls $OMNIBUS_PACKAGE_DIR
+  only:
+    - master
+    - tags
+  tags: [ "runner:main", "size:large" ]
+  script:
+    - source /usr/local/rvm/scripts/rvm
+    - rvm use 2.4
```

Comment:
let's strive to make our pipelines as fast as possible, that means not installing dependencies everytime, when we can ship them in the builder image.
[We already have 2.3](https://github.com/DataDog/datadog-agent-builders/blob/master/deb-x64/Dockerfile#L22), could either omnibus run with 2.4 or test-kitchen run in 2.3, instead of installing another version?

Reply:
This is not installing it, it's just using it. It's installed on the docker image I'm creating for this.